easybuild/easyconfigs/e/enable/enable-20160817_kiva.patch

```diff
@@ -0,0 +1,29 @@
+#non-git version (without .git directory) build fails
+#bundled kiva version is tied to enable
```

Comment:
ok, please extend the comment a bit to reflect this better? e.g. include the URL, etc?